Density/app/src/main/java/org/cornelldti/density/density/facilitydetail/feedback/FeedbackFirstFragment.kt

```diff
@@ -0,0 +1,82 @@
+package org.cornelldti.density.density.facilitydetail.feedback
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Button
+import android.widget.ImageView
+import android.widget.RadioGroup
+import androidx.fragment.app.Fragment
+import androidx.localbroadcastmanager.content.LocalBroadcastManager
+import androidx.viewpager2.widget.ViewPager2
+import org.cornelldti.density.density.R
+
+
+class FeedbackFirstFragment : Fragment() {
+
+    private lateinit var viewPager: ViewPager2
+    private lateinit var radioGroup : RadioGroup
+    private lateinit var buttonNext : Button
+    private lateinit var buttonClose : ImageView
+    private var selectedAnswer = 0
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
+                              savedInstanceState: Bundle?): View? {
+        return inflater.inflate(R.layout.fragment_feedback_first, container, false)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        viewPager = parentFragment?.view?.findViewById(R.id.viewPager) ?: context?.let { ViewPager2(it) }!!
+        radioGroup = view.findViewById(R.id.answer_dialog_radio)
+        buttonNext = view.findViewById(R.id.button_next)
+        buttonClose = view.findViewById(R.id.button_close)
+
+        setRadioGroup()
+        setButtonNext()
+        setButtonClose()
+    }
+
+    private fun setRadioGroup(){
+        radioGroup.setOnCheckedChangeListener { radioGroup, i ->
+            if (i==R.id.answer_yes){
```

Comment:
Some spacing would be nice

Reply:
for future reference, there's an [auto-format shortcut in Android Studio](https://stackoverflow.com/a/16580200/3308746) - we should probably implement a pre-commit action for formatting/linting